Demo/Demo/ViewModels/CardPaymentViewModel.swift

```diff
@@ -10,14 +10,22 @@ class CardPaymentViewModel: ObservableObject, CardDelegate {
 
     private var cardClient: CardClient?
 
-    func createOrder(amount: String, selectedMerchantIntegration: MerchantIntegration, intent: String) async throws {
+    func createOrder(
+        amount: String,
+        selectedMerchantIntegration: MerchantIntegration,
+        intent: String,
+        isVaulting: Bool,
+        customerID: String? = nil
+    ) async throws {
         // might need to pass in payee as payee object or as auth header
 
         let amountRequest = Amount(currencyCode: "USD", value: amount)
         // TODO: might need to pass in payee as payee object or as auth header
         let orderRequestParams = CreateOrderParams(
             intent: intent,
-            purchaseUnits: [PurchaseUnit(amount: amountRequest)]
+            purchaseUnits: [PurchaseUnit(amount: amountRequest)],
+            isVaulting: isVaulting,
```

Comment:
```suggestion
            shouldVault: shouldVault,
```
🧶 : personal style, i've always liked "should" when paired with vault. I feel like it bucks strict conventions, but it reads well and serves the same purpose as "is" in regards to the "Vault" feature.

Reply:
Addressed in e2f1c5967aa90e94f758493b70ef36e2732e78b3